open3d_slam_ros/src/mapping_node.cpp

```diff
@@ -15,12 +16,19 @@ int main(int argc, char **argv) {
 	ros::NodeHandlePtr nh(new ros::NodeHandle("~"));
 
 	const bool isProcessAsFastAsPossible = nh->param<bool>("is_read_from_rosbag", false);
+	const bool initializeMap = nh->param<bool>("initialize_map", false);
```

Comment:
We need to see where to put these flags. Right now they are a bit spread through the launch file and the yaml file, which can create confusion. I like using the launch file to define paths to bags and config files since we can just use ($find _ros_package_name_), but maybe it would be better to keep all the algorithm-specific stuff in the yaml file. (same also goes for _is_read_from_rosbag_, but don't worry about that one). Consider moving _initialize_map_ to the .yaml file to the mapping section next to
 (is_print_timing_information: true
  is_build_dense_map: false
  is_attempt_loop_closures: true) 

I would also rename _initialize_map_ to _is_initialize_map_ to keep the naming consistant.